lib/fog/kubevirt/compute/requests/get_vm.rb

```diff
@@ -12,11 +12,72 @@ def get_raw_vm(name)
       end
 
       class Mock
-        # TODO provide implementation
         def get_vm(name)
+          Vm.parse get_raw_vm(name)
         end
 
         def get_raw_vm(name)
+          {
+            :apiVersion=>"kubevirt.io/v1alpha3",
+            :kind=>"VirtualMachine",
+            :metadata=>{
+              :creationTimestamp=>"2019-03-21T10:44:20Z",
+              :generation=>1,
+              :labels=>{
+                :"kubevirt.io/vm"=>"test"
+              },
+              :name=>"test",
+              :namespace=>"default",
+              :resourceVersion=>"734888",
+              :selfLink=>"/apis/kubevirt.io/v1alpha3/namespaces/default/virtualmachines/test",
+              :uid=>"489c6fba-4bc6-11e9-9c46-bce22353a505"
+            },
+            :spec=>{
+              :running=>false,
+              :template=>{
+                :metadata=>{
+                  :creationTimestamp=>nil,
+                  :labels=>{
+                    :"kubevirt.io/vm"=>"test"
+                  }
+                },
+                :spec=>{
+                  :domain=>{
+                    :cpu=>{
+                      :cores=>1
+                    },
+                   :devices=>{
+                     :disks=>[{
```

Comment:
Please include also 2 network interfaces with mac address, one for pod network and other for multus, to make this one more complete.